.travis.yml.bak

```diff
@@ -29,13 +29,13 @@ jobs:
 
     # this job do deploy maven repository
     # unit tests (openjdk11)
-    - jdk: openjdk11
-      env:
-        - DESC="tests and deploy"
-        - CMD="mvn -e --no-transfer-progress clean integration-test failsafe:verify
-          -DargLine='-Xms1024m -Xmx2048m'"
-        - DEPLOY="true"
-        - USE_MAVEN_REPO="true"
+#    - jdk: openjdk11
+#      env:
+#        - DESC="tests and deploy"
+#        - CMD="mvn -e --no-transfer-progress clean integration-test failsafe:verify
+#          -DargLine='-Xms1024m -Xmx2048m'"
+#        - DEPLOY="true"
```

Comment:
This job is fancy.
This variable trigger post build execution  to deploy to maven snapshot public repository, by secret credentials.
Better to keep it Travis.

Reply:
It is ok to disable or keep it failing.

Reply:
Done, it will be the only job left in Travis now.